src/Okta.Sdk/IPoliciesClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿// <copyright file="IPoliciesClient.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Okta.Sdk
+{
+    /// <summary>A client that works with Okta Policies resources.</summary>
+    public partial interface IPoliciesClient
```

Comment:
What about https://developer.okta.com/docs/api/resources/policy#get-all-policies-by-type ?

Reply:
I saw `ListPolicies` used earlier, but it doesn't seem to be included here.

Reply:
@nbarbettini take a look at `GetAllPoliciesByType` which covers this case.